content/docs/intro/install.md

```diff
@@ -52,7 +52,7 @@ You can fetch that script, and then execute it locally. It's well documented so
 that you can read through it and understand what it is doing before you run it.
 
 ```console
-$ curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 > get_helm.sh
+$ curl -o get_helm.sh -f -s -S -L https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
```

Comment:
You probably don't want to fail silently (`-f`) if the goal is for users to more easily detect potential download errors 😄 

Can also be shortened:

```suggestion
$ curl -sSLo get_helm.sh "https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3"
```

Reply:
I shortened the command, but I left the `-f` in there.

If you do something like:

```
curl -fsSLo get_helm.sh "https://raw.githubusercontent.com/helm/helm/master/scripts/does-not-exist"
```

You only get this error on stderr:
```
curl: (22) The requested URL returned error: 404 Not Found
```